apps/system/index.html

```diff
@@ -274,6 +274,7 @@
     <!-- Notifications -->
     <link rel="stylesheet" type="text/css" href="style/notifications/notifications.css">
     <script defer src="js/notifications.js"></script>
+    <script defer src="js/notifications_system_message.js"></script>
```

Comment:
You don't need this - Core will load it for you since you defined it as submodule of Core.